src/github/prComment.ts

```diff
@@ -225,6 +235,16 @@ export class GHPRComment implements vscode.Comment {
 		this.timestamp = new Date(comment.createdAt);
 	}
 
+	get body(): vscode.MarkdownString | string {
+		// VS Code's markdown rendering is more correct and will not render single line breaks as
+		// line breaks in markdown. To make the comment look more like github.com, we replace single line breaks with double.
+		return (this.mode === vscode.CommentMode.Editing) ? this.rawBody : new vscode.MarkdownString(this.rawBody.replace(/(?<!\s)(\r\n|\n)(?!\s)/g, '\n\n'));
```

Comment:
Why is this a different regex than above?
Does it make sense to extract this into a new function?

Reply:
Thanks for the reminder! I prefer to do refactors separate from other changes: https://github.com/microsoft/vscode-pull-request-github/pull/3921